src/component/Registration/SignIn.css

```diff
@@ -0,0 +1,66 @@
+.registrationForm {
+    background-color: #fff;
+    color: black;
```

Comment:
Let's bee consistent and stick with hex codes

Reply:
corrected boss